azure/iac/networkwatchersflowlogs.rego

```diff
@@ -14,6 +14,12 @@ azure_attribute_absence["netwatch_logs"] {
     not resource.properties.enabled
 }
 
+azure_attribute_absence["netwatch_logs"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.network/networkwatchers/flowlogs"
+    not resource.properties.flowAnalyticsConfiguration.networkWatcherFlowAnalyticsConfiguration.enabled
+}
+
```

Comment:
`resource.properties.flowAnalyticsConfiguration.networkWatcherFlowAnalyticsConfiguration.enabled`

This should be a sperate rule from `resource.properties.enabled` as its optional parameter. someone can just create the `microsoft.network/networkwatchers/flowlogs` without` resource.properties.flowAnalyticsConfiguration` unless we enforce as a good practice.
@farchide @shahinshirvani

Reply:
Created separate rule for each one of them and updated remediation